server/chat-plugins/youtube.ts

```diff
@@ -429,9 +429,8 @@ export const pages: PageTable = {
 				return this.errorReply(`There are currently no categories in the Youtube channel database.`);
 			}
 			const sorted: {[k: string]: string[]} = {};
-			const channels = Object.assign({}, YouTube.data.channels);
-			for (const id of Object.keys(channels)) {
-				const channel = channels[id];
+			const channels = YouTube.data.channels;
+			for (const [id, channel] of Object.entries(channels)) {
```

Comment:
@mia-pi-git Do you remember why you used `Object.assign` here? `channels` doesn't seem to be modified ever.

Reply:
I... don't remember doing this at all. Should be fine to remove.